src/metabase/api/database.clj

```diff
@@ -353,36 +352,34 @@
   []
   (saved-cards-virtual-db-metadata :card :include-tables? true, :include-fields? true))
 
-(defn- filter-tables-by-data-model-perms
-  [tables]
-  (if-let [f (u/ignore-exceptions
-              (classloader/require 'metabase-enterprise.advanced-permissions.common)
-              (resolve 'metabase-enterprise.advanced-permissions.common/filter-tables-by-data-model-perms))]
-    (f tables)
-    tables))
```

Comment:
This function can be removed because `filter-databases-by-data-model-perms` recursively filters tables, and that's already being called in `db-metadata`, so this was superfluous